examples/ui-demo/src/components/shared/mint-card/MintCardActionButtons.tsx

```diff
@@ -0,0 +1,53 @@
+import { cn } from "@/lib/utils";
+
+type MintCardActionButtonsProps = {
+  nftTransfered: boolean;
+  handleCollectNFT: () => void;
+  status: Record<string, string>;
+  transactionUrl: string;
+} & React.HTMLAttributes<HTMLDivElement>;
+
+export function MintCardActionButtons({
+  nftTransfered,
+  handleCollectNFT,
+  status,
+  transactionUrl,
+  className,
+  ...props
+}: MintCardActionButtonsProps) {
+  return (
+    <div {...props} className={cn("px-6 xl:px-0", className)}>
+      {!nftTransfered ? (
+        <button
+          className="btn btn-primary w-full p-2 radius mb-4 xl:mb-4"
+          disabled={Object.values(status).some((x) => x === "loading")}
```

Comment:
nit: I would pass disabled through to here as well if you're going to pass the `handleCollect` as well. Right now there's a tight coupling between this and its parent since this component now has to know what type of statuses can come through (exacerbated by the `Record<string, string>` typing which provides no hints to this).

In https://github.com/alchemyplatform/aa-sdk/pull/1020, I actually made the Mint Card show up regardless of client != null or not as long as the user isn't EOA and set the button to disabled to prevent the flashes of content on refreshes. Having this be a `disabled` prop will make it easier to merge these 2 PRs together